lib/assets/javascripts/unpoly/form.coffee.erb

```diff
@@ -198,6 +198,8 @@ up.form = do ->
     If set to `'auto'` (default), Unpoly will try to find a match in the form's layer.
   @param {string} [options.failLayer='auto']
     The name of the layer that ought to be updated if the server sends a non-200 status code.
+  @param {up.Params} [options.params=up.Params.fromForm(form)]
```

Comment:
The code below will already work with other types of params:

```
@param {Object|FormData|string|Array|up.Params}
```

Reply:
Sure! I've changed it in the docblock